software/speller/data_collection_platform/backend/dcp/__init__.py

```diff
@@ -43,7 +43,7 @@ def create_app():
         migrate.init_app(app, db)
 
         # Added blueprint for clearing the database
-        from .clear_database import usersbp
-        app.register_blueprint(usersbp)
+        from .commands import commands_bp
```

Comment:
let's not use relative imports here, 
```suggestion
        from dcp.commands import commands_bp
```